x_offset: -0.01
y_offset: 0.027
z_offset: 0.035
roll_offset: 5
pitch_offset: 0
yaw_offset: 0
hip_pitch_offset: 12.0
period_time: 600
dsp_ratio: 0.2
step_forward_back_ratio: 0.28
foot_height: 0.041
swing_right_left: 0.025
swing_top_down: 0.006
pelvis_offset: 1
arm_swing_gain: 0.8
balance_hip_roll_gain: 0.5
balance_knee_gain: 0.2
balance_ankle_roll_gain: 0.35
balance_ankle_pitch_gain: 0.35
p_gain: 0
i_gain: 0
d_gain: 0
